Test/Test.cs

```diff
@@ -55,10 +55,16 @@ public void Setup()
             CreateDocResponse = JsonSerialPublisher.ObjectToDict<Dictionary<string, object>>(response);
         }
 
-        [Test]
-        public void TestCreateAsset() {
+        [TestCase("name", "description")]
+        [TestCase("", "")]
+        // [TestCase(null, null)] TODO: figure out why this doesn't work
```

Comment:
For CreateAsset there is a flaw in the API such that it doesn't accept null for name and description. For the others Create and Patch functions that use name and description it should be OK.